1.0/docs/devguide/properties.md

```diff
@@ -613,6 +613,18 @@ Example:
       </script>
     </dom-module>
 
+#### Using native array mutation methods {#notifysplices}
+
+Whenever possible you should always use Polymer's 
+[array mutation methods](#array-mutation).
+However, in some scenarios you might be forced to mutate arrays using the native
+`Array.prototype` methods. For example, you may be using a third-party library
```

Comment:
This is a little wordy. What do you think of just:

"However, this isn't always possible." for the 2nd sentence here?

And maybe "that is not aware of" => "that does not use"?